.github/workflows/cd.yml

```diff
@@ -1,5 +1,7 @@
 name: Continuous Delivery
 on:
+  merge_group:
```

Comment:
I think we don't need to have it here since this one only runs after stuff is merged

Reply:
Good catch, fixed.